infrastructure/templates/public-api/parameters/main-preprod.bicepparam

```diff
@@ -3,6 +3,11 @@ using '../main.bicep'
 // Environment Params
 param environmentName = 'Pre-Production'
 
+param publicUrls = {
+  contentApi: 'https://s101p02-as-ees-content.azurewebsites.net'
```

Comment:
Is it not possible to use `https://content.pre-production.explore-education-statistics.service.gov.uk` here?

Reply:
For some reason, this doesn't have a working SSL certificate. I'm not sure about the context surrounding this, but someone else might know.

For now, I've just used this URL as it's also the one being used in the ARM template `pre-prod.json` parameters as well.

Reply:
Cool, I wasn't aware of the certificate warning. Thanks. 🙂